manifests/charts/ztunnel/files/profile-platform-openshift.yaml

```diff
@@ -17,3 +17,5 @@ seLinuxOptions:
   type: spc_t
 # Openshift requires privileged pods to run in kube-system
 trustedZtunnelNamespace: "kube-system"
+platform: "openshift"
```

Comment:
Hello @dgn , I checked more in the manifests. I misunderstood the values.platform field. This one is not necessary.
Because automatic generated values have already included this one .
Reference: https://github.com/istio/istio/blob/master/manifests/zzz_profile.yaml#L42

I will remove this replicated one in my PR. Thanks.